src/styles/watch.scss

```diff
@@ -0,0 +1,41 @@
+.watch {
+  position: relative;
+  @include square($watch-size);
+  border: 1px black solid;
+  margin-left: #{$watch-shift};
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.